### PR TITLE
6 variable border color

### PR DESCRIPTION
**Pull request by nyaray**
A pretty sweet diff, if I might say so myself.

### CODE REVIEW

File: lib/hal/hal.ex [hunk 1]
```diff
@@ -7,7 +7,8 @@ defmodule Inky.HAL do
 
   @callback init(opts :: map()) :: io_state()
   @callback handle_update(
-              opts :: map(),
+              pixels :: map(),
+              border :: :white | :black | :red | :yellow | :accent,
```
Comment (lawik) [inline lib/hal/hal.ex:11]: Are we always accepting `:red | :yellow | :accent` where we accept color? I thought it was white, black, accent only. Should it be?

File: lib/hal/rpihal.ex [hunk 2]
```diff
@@ -2,8 +2,9 @@ defmodule Inky.RpiHAL do
   @default_io_mod Inky.RpiIO
 
   @moduledoc """
-  An `Inky.HAL` implementation responsible for sending commands to the Inky screen. It delegates
-  to whatever IO module its user provides at init, but defaults to #{inspect(@default_io_mod)}
+  An `Inky.HAL` implementation responsible for sending commands to the Inky
+  screen. It delegates to whatever IO module its user provides at init, but
+  defaults to #{inspect(@default_io_mod)}
```
Comment (lawik) [inline lib/hal/rpihal.ex:7]: Dark magic, you can inject module attributes into docs. I guess its fine though it is a bit weird for in-code readability :)

File: lib/hal/rpihal.ex [hunk 3]
```diff
@@ -30,7 +31,7 @@ defmodule Inky.RpiHAL do
 
   @impl HAL
   def init(args) do
-    display = Map.fetch!(args, :display)
+    display = args[:display] || raise(ArgumentError, message: ":display missing")
```
Comment (lawik) [inline lib/hal/rpihal.ex]: Is the message based off of any particular pattern? Seems terse.
":display missing in init args" or something to that effect if we need a custom error at all would be my first guess. But  haven't raised a lot so far.

**Review by lawik — COMMENTED** [review]
If you are happy in spite of my delicious notes. Feel free to merge. Otherwise, fix my delicious notes as far as you feel like :)

File: lib/hal/hal.ex (hunk 1 above)
Comment (nyaray) [reply lib/hal/hal.ex:11]: We accept all colors, https://github.com/pappersverk/inky/blob/master/lib/hal/rpihal.ex#L12, and have done for quite some time. 🤷‍♂

File: lib/hal/rpihal.ex (hunk 2 above)
Comment (nyaray) [reply lib/hal/rpihal.ex:7]: ⭐️ Teachable moment ⭐️ 

(module)docs are just strings that are set as module attributes and later interpreted in special ways. This is just regular string interpolation ^^

File: lib/hal/rpihal.ex (hunk 3 above)
Comment (nyaray) [reply lib/hal/rpihal.ex]: Hm. Good point about the message, I'll add `in init args`.

File: lib/hal/hal.ex (hunk 1 above)
Comment (lawik) [reply lib/hal/hal.ex:11]: Ok, cool. But only for very small values of all ;)

File: lib/hal/rpihal.ex (hunk 2 above)
Comment (lawik) [reply lib/hal/rpihal.ex:7]: Oh, sure. I understood the string interpolation and figured it was a normal string. I hadn't considered it had access to the module context :)

**Pull request merged**